admin/tabs/activity/tracking/detail/ActivityDetailModel.ts

```diff
@@ -9,15 +9,14 @@ import * as Col from '@xh/hoist/admin/columns';
 import {FormModel} from '@xh/hoist/cmp/form';
 import {GridModel} from '@xh/hoist/cmp/grid';
 import {HoistModel, lookup, managed} from '@xh/hoist/core';
-import {fmtJson} from '@xh/hoist/format';
 import {action, computed, makeObservable, observable} from '@xh/hoist/mobx';
 import {ActivityTrackingModel} from '../ActivityTrackingModel';
 
 export class ActivityDetailModel extends HoistModel {
     @lookup(ActivityTrackingModel) activityTrackingModel: ActivityTrackingModel;
     @managed gridModel: GridModel;
     @managed formModel: FormModel;
-    @observable formattedData;
+    @observable displayData;
```

Comment:
This model is no longer doing the formatting. Update the name to reflect. See https://github.com/xh/hoist-react/pull/3937/files#diff-2a82d78cc59aadd90860fcd0383be76d3b7491d7b19736a18857b6b93d30cd66L117-L125